static/js/overlays.js

```diff
@@ -224,6 +224,16 @@ exports.initialize = function () {
 
         const target_name = $target.attr("data-overlay");
 
+        if (target_name === 'subscriptions') {
+            const sub = subs.active_stream();
+            if (sub !== undefined) {
+                const colorpicker = $("#subscription_overlay .subscription_settings[data-stream-id='" + sub.id + "'] .colorpicker");
+                if (colorpicker.parent().find('.sp-replacer').hasClass('sp-active')) {
+                    colorpicker.spectrum('hide');
+                }
+            }
+        }
+
```

Comment:
I think it might be cleaner to include in the `subs.js` handler for closing its overlay the logic to unconditionally hide any colorpickers inside?  I'd like to keep the `overlays.js` code clear of custom per-overlay code like this.